src/cdk-experimental/selection/selection-set.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {TrackByFunction} from '@angular/core';
+import {Subject} from 'rxjs';
+
+interface TrackBySelection<T> {
```

Comment:
Could you add a JsDoc description for this? It's not exported, but it would make it a bit more clear for internal maintenance